### PR TITLE
Add ballholder component

### PR DESCRIPTION
**Pull request by marcioflaviof**

### CODE REVIEW

File: src/components/molecules/BallHolder.tsx [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import Ball from "../atom/Ball";
+
+import "./style.css";
+
+const BallHolder = () => {
```
Comment (wenderjean) [inline src/components/molecules/BallHolder.tsx]: What do you think about naming it as `Results`?? It sounds clearer in the domain of the application.

File: src/components/molecules/style.css [hunk 2]
```diff
@@ -0,0 +1,4 @@
+.ball-holder {
+    display: flex;
+    flex-flow: row wrap;
+}
```
Comment (wenderjean) [inline src/components/molecules/style.css]: Do you have your editor configured regarding line-endings?

File: src/components/molecules/style.css [hunk 3]
```diff
@@ -0,0 +1,4 @@
+.ball-holder {
+    display: flex;
```
Comment (wenderjean) [inline src/components/molecules/style.css]: Should we use 2 spaces to indent?

**Review by wenderjean — APPROVED** [review]

File: src/components/molecules/style.css (hunk 2 above)
Comment (marcioflaviof) [reply src/components/molecules/style.css]: Yes, but a forgot to format again, already changed my configs to format on save now

File: src/components/molecules/Results/Results.tsx [hunk 4]
```diff
@@ -0,0 +1,18 @@
+import Ball from "../atom/Circle";
+
+import "./style.css";
+
+const Results = () => {
```
Comment (wenderjean) [inline src/components/molecules/Results/Results.tsx]: Let's receive the results as an array and render all of them, it'll make your component flexible.

**Pull request merged**